sal_interfaces/DIMM/DIMM_Events.xml

```diff
@@ -50,6 +50,20 @@
       <Units>second</Units>
       <Count>1</Count>
     </item>
+    <item>
+      <EFDB_Name>expiresAt</EFDB_Name>
+      <Description>Timestamp until when the measurement is deemed valid.</Description>
```

Comment:
do you want to add that it's in unix_tai ? might save headaches for people later.